### DISCUSSION TITLE
Looking forward to using pynecone

### DISCUSSION THREAD

**Discussion by gsal** [reflex-dev/reflex#124] [General]
Hi, everyone: 

I started to explore the web-framework world, for the first time ever, just about 2 weeks ago; not knowing JS, nor caring for HTML/ CSS but knowing Python, I was happy to find pure-python frameworks.  So, I am exploring a few, like Flet, Panel, Bokeh, Streamlit, NiceGUI. Pynecone being a "newborn baby", there are no reviews nor comparisons vs existing frameworks...I don't even remember how I found it, but I like it

Clearly, each framework has its own approach, its own idiosyncracies; so, far, I think I like Pynecone approach and I believe I understand it, but I need to get my feet wet.

Anyway, looking forward to using it.  

Thanks.

**Comment by Alek99** [comment]
Thanks! Hope you like it, let me know if you have any questions

**Comment by Alek99** [comment]
Banned that user who posted that content, I don't know what's happening today lots of bots on github.

**Comment by Alek99** [comment]
Another comment they have been banned aswell.
